app/public/data/data.json

```diff
@@ -1,5 +1,21 @@
 {
-  "boards": [],
+  "boards": [
+    [
+      {
+        "name": "bratmooj"
+      }
+    ],
+    [
+      {
+        "name": "bratmooj"
+      }
+    ],
+    [
+      {
+        "name": "bratmooj"
+      }
```

Comment:
Zasto 3 board-a sa istim imenom?

Reply:
evo ne znam bogami to je sa predavanja xd